deepchem/models/tests/test_overfit.py

```diff
@@ -936,16 +895,12 @@ def test_tf_progressive_regression_overfit(self):
         dropouts=[0.],
         learning_rate=0.003,
         weight_init_stddevs=[.1],
-        seed=123,
         alpha_init_stddevs=[.02],
         batch_size=n_samples)
 
     # Fit trained model
     model.fit(dataset, nb_epoch=20)
-    model.save()
```

Comment:
Need to mention that progressive model currently cannot be saved, still trying to dig into it but I suspect the submodel setting might be the cause.

Reply:
@miaecle Would you mind raising a new issue documenting the error? This sounds like a general problem we should fix. We can do this in a different PR though.